jobserver/settings.py

```diff
@@ -161,6 +161,28 @@
 
 # Insert Whitenoise Middleware.
 STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
+WHITENOISE_SKIP_COMPRESS_EXTENSIONS = (
+    "jpg",
+    "jpeg",
+    "png",
+    "gif",
+    "webp",
+    "zip",
+    "gz",
+    "tgz",
+    "bz2",
+    "tbz",
+    "xz",
+    "br",
+    "swf",
+    "flv",
+    "woff",
+    "woff2",
+    "svg",
+    "ico",
+    "webmanifest",
+    "json",
```

Comment:
Can you order these alphabetically please?  It'll be easier to see what we're using when we need to add another one later.

Reply:
Good point, done ✅